tap_urban_airship/__init__.py

```diff
@@ -114,7 +115,7 @@ def flatten_channels(item):
         item['channels'] = [c['channel_id'] for c in item['channels']]
         return item
 
-    sync_entity("named_users", ["named_user_id"], transform=flatten_channels)
+    sync_entity("named_users", ["named_user_id"], ["created", "last_modified"], transform=flatten_channels)
```

Comment:
It appears that these fields are [undocumented](https://docs.urbanairship.com/api/ua/#schemas/nameduserresponsebody). If that's the case then we at a minimum need to add a comment here to that effect. I think it would also be prudent to add a `raise` in the event that our expectation that these fields are always found is violated. I think that would be handled by #7 so we should get that merged first.

Reply:
This is what is being returned from Urban Airship API here:
```
{
  "named_user_id": "...",
  "tags": {...},
  "created": "...",
  "last_modified": "...",
  "channels": [
    {...}
  ]
}
```
So you are right, it doesn't match the API documentation. I will add a comment in the code.
And yes, the code that handles the situation when date fields are not present is in PR #7.

Reply:
@timvisher Added a comment, please check it out.